src/getInputValues.js

```diff
@@ -0,0 +1,33 @@
+import { exceptionCarsNameInput, exceptionRacingCountInput } from './inputExceptions.js';
+import { Car } from './Car.js';
+
+const alertMsg = {
+  msg: '다시 입력해주세요',
+};
+
+const getCarsNameInput = () => {
+  const EARLY_DISTANCE = 0;
+  const carsNameInput = document.querySelector('#car-names-input').value;
+
+  if (exceptionCarsNameInput(carsNameInput) === false) {
+    alert(alertMsg.msg);
+    document.querySelector('#car-names-input').value = '';
+  } else {
+    const $carsNameInput = carsNameInput.split(',')
+      .map((e) => new Car(e, EARLY_DISTANCE));
+    return $carsNameInput;
+  }
```

Comment:
얼리리턴을 사용해주시면 가독성 측면에서 더 좋을 것 같아요! 
```suggestion
  if (exceptionCarsNameInput(carsNameInput) === false) {
    alert(alertMsg.msg);
    document.querySelector('#car-names-input').value = '';
    return;
  }
  
  const $carsNameInput = carsNameInput.split(',')
      .map((e) => new Car(e, EARLY_DISTANCE));
  return $carsNameInput;
```

Reply:
맞습니다! 어제 공부하다가 return 하면 그 뒤는 진행되지 않지! 하고 깨달은 부분입니다. 다시 한 번 일깨워주셔셔 감사드립니다!